### DISCUSSION TITLE
zh/config/

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#45] [Announcements]
# zh/config/

AList v3的文档

https://alist.nn.ci/zh/config/

**Comment by at86397388** [comment]
请问我设置好后，如果mycloud gen2重启的话，之前的设置就被初始化了，如果我重启NAS后，可以通用配置文件的形式运行么？这样配置就不会被初始化了，如果可以是用哪个命令呢？配置文件又是放在哪里的呢，谢谢

**Reply by at86397388** [reply]
已找到

**Reply by at86397388** [reply]
但如何通过配置文件启动还不会

**Comment by kaixin1588** [comment]
密码忘记了
，怎么办

**Comment by pzj37** [comment]
不会登录
